front/src/pages/_app.js

```diff
@@ -1,6 +1,5 @@
 import "@/styles/globals.css"
-import { appWithTranslation } from "next-i18next"
 
-const App = ({ Component, pageProps }) => <Component {...pageProps} />
-
-export default appWithTranslation(App)
+export default function App({ Component, pageProps }) {
```

Comment:
Lack of consistency between this and say for example `const Header = () => ...`.

Reply:
It was a revert that's why it goes back to a `function` instead of a `const`

Reply:
The auto merge didn't let me checked the branch before merging into staging so staging was broken